Sources/PackageDescription/Target.swift

```diff
@@ -432,13 +486,65 @@ public final class Target {
             pkgConfig: pkgConfig,
             providers: providers)
     }
+
+    /// Create a binary target referencing a remote artifact.
+    ///
+    /// A binary target provides the url to a pre-built binary artifact for the target. Currently only supports
+    /// artifacts for Apple platforms.
+    ///
+    /// - Parameters:
+    ///   - name: The name of the target.
+    ///   - url: The URL to the binary artifact. Should point to a `zip` archive containing a `XCFramework`.
+    ///   - checksum: The checksum of the artifact archive for validation.
+    @available(_PackageDescription, introduced: 5.2)
```

Comment:
Since this won't be going to 5.2, we should make sure this is properly annotated once we know the next swift release. Can you add a FIXME here and then file a JIRA so we don't miss it?